UPGRADE-6.7.md

```diff
@@ -2465,6 +2467,19 @@ After:
 <mt-checkbox @update:checked="updateValue" />
 ```
 </details>
+
+### Deprecated admin notification entity + related classes
+
+We have moved the notification entity, collection and definition to core. You should update your code to reference the new classes:
```

Comment:
do we need to mention the old class names?

Reply:
added